Kernel/Memory/MemoryManager.h

```diff
@@ -49,15 +49,13 @@ inline FlatPtr virtual_to_low_physical(FlatPtr virtual_)
 
 enum class UsedMemoryRangeType {
     LowMemory = 0,
-    Prekernel,
     Kernel,
     BootModule,
     PhysicalPages,
 };
 
 static constexpr StringView UserMemoryRangeTypeNames[] {
```

Comment:
Should we `static_assert` the size of this array matches the `UsedMemoryRangeType` enum?